src/pam/oslogin_sshca.cc

```diff
@@ -308,25 +307,23 @@ static int GetByoidFingerPrint(pam_handle_t *pamh, const char *blob, char **fing
   return f_len;
 }
 
-int FingerPrintFromBlob(pam_handle_t *pamh, const char *blob, char **fingerprint) {
+int FingerPrintFromBlob(const char *blob, char **fingerprint) {
   char *line, *saveptr = NULL;
   size_t f_len = 0;
 
   if (blob == NULL || strlen(blob) == 0) {
-    PAM_SYSLOG(pamh, LOG_ERR, "Could not parse/extract fingerprint "
-               "from ssh ca cert's extension: \"blob\" is empty.");
+    SysLogErr("Could not parse/extract fingerprint from ssh ca cert's extension: \"blob\" is empty.");
     return 0;
   }
 
   if (fingerprint == NULL) {
-    PAM_SYSLOG(pamh, LOG_ERR, "Could not parse/extract fingerprint "
-               "from ssh ca cert's extension: \"fingerprint\" is NULL.");
+    SysLogErr("Could not parse/extract fingerprint from ssh ca cert's extension: \"fingerprint\" is NULL.");
```

Comment:
suggestion: "SSH CA" instead of "ssh ca"

Reply:
updated. thanks.